randovania/games/cave_story/logic_database/Last Cave.txt

```diff
@@ -352,6 +347,27 @@ Extra - in_dark_aether: False
   > Exit to Passage?
       Trivial
 
+> After Spike Pit; Heals? False
+  * Layers: default
+  > H/V Trigger to Sacred Grounds - B2
+      All of the following:
+          Has Weapon
+          Any of the following:
+              Has Flight
+              # Damage boost off of spikes
+              Damage Boosting (Beginner) and Normal Damage ≥ 10
+  > Pickup (Hell B3 Chest)
+      All of the following:
+          Has Flight
+          Any of the following:
+              Can Break Blocks
+              Bubbler and Break blocks with Bubbler (Beginner)
+              All of the following:
+                  Missiles and Break blocks with Missiles (Beginner)
+                  Missile Launcher or Super Missile Launcher
```

Comment:
The DB has this formatted a tad weirdly:
![grafik](https://github.com/randovania/randovania/assets/38186597/8de55587-1e36-4604-b2b7-d5e6a8eb8de0)


I don't consider it blocking, but it would be nice for the DB to look like the pretty-print here.

Reply:
as far as I'm concerned that's a bug far beyond the scope of this pr

Reply:
The pretty print has an additional `simplify`, but the database doesn't for the sake of being precise.